zap/src/main/java/org/parosproxy/paros/extension/history/HistoryTable.java

```diff
@@ -34,7 +34,14 @@ public HistoryTable() {
         setAutoCreateColumnsFromModel(false);
 
         setName("History Table");
+        super.loadColumnConfiguration("HistoryTable");
+        if (this.config == null) {
```

Comment:
This check could be done in `loadColumnConfiguration`, less checks needed by extending classes and we could also remove the instance variable `config` (which seems to be used only to signal that the persistence was previously done).